src/wagtail_localize_smartling/settings.py

```diff
@@ -27,6 +31,9 @@ class SmartlingSettings:
         default_factory=dict
     )
     REFORMAT_LANGUAGE_CODES: bool = True
+    JOB_DESCRIPTION_CALLBACK: (
+        Callable[[str, "TranslationSource", Iterable["Translation"]], str] | None
```

Comment:
My type annotation is rusty - here is this staying the callable takes three params (str, TranslationSource, Translation) and returns a string, or is there one extra param here (the final str) that shouldn't be?

Reply:
That is correct (ref: https://peps.python.org/pep-0677/#an-arrow-syntax-for-callable-types)

`Callable[var1, var2]` is fn with 2 params. `Callable[[var1, var2], output]` is the fn with the 2 params and the output type.